extensions/common/iam/identity-trust/identity-trust-transform/src/main/java/org/eclipse/edc/iam/identitytrust/transform/to/JsonObjectToVerifiableCredentialTransformer.java

```diff
@@ -84,7 +86,21 @@ private Instant parseDate(JsonValue jsonValue, TransformerContext context) {
         return Instant.parse(str);
     }
 
-    private Object parseIssuer(JsonValue jsonValue, TransformerContext context) {
-        return transformString(jsonValue, context); //todo: handle the case where the issuer is an object
+    private Issuer parseIssuer(JsonValue jsonValue, TransformerContext context) {
+        if (jsonValue.getValueType() == JsonValue.ValueType.STRING) {
+            return new Issuer(transformString(jsonValue, context), Map.of());
+        } else {
+            // issuers can be objects, that MUST contain an ID, and optional other properties
+            // an issuer is never an array with >1 elements
+            JsonObject issuer;
+            if (jsonValue.getValueType() == JsonValue.ValueType.ARRAY) {
+                issuer = jsonValue.asJsonArray().get(0).asJsonObject();
```

Comment:
Don't we need to check the array is not empty before doing that? 
Might be a sneaky (and probably very unlikely) case in practice, but to be sure :)

Reply:
Assuming we have validation on the ingress, we can assume that the issuer (mandatory field) is there, and since the jsonld is expanded, the array can't be empty

Reply:
fair enough